x/ibc/02-client/exported/exported.go

```diff
@@ -215,3 +217,61 @@ func ClientTypeFromString(clientType string) ClientType {
 		return 0
 	}
 }
+
+// Height is an interface for a monotonically increasing data type
+// that can be compared against for the purposes of updating and freezing clients
+// Compare implements a method to compare two heights. When comparing two heights a, b
+// we can call a.Compare(b) which will return
+// negative if a < b
+// zero     if a = b
+// positive if a > b
+//
+// Decrement will return a decremented height from the given height. If this is not possible,
+// an error is returned
+// Valid returns true if height is valid, false otherwise
+type Height struct {
+	EpochNumber uint64 `json:"epoch_number" yaml:"epoch_number"`
+	EpochHeight uint64 `json:"epoch_height" yaml:"epoch_height"`
```

Comment:
maybe for a third party, it's not so clear what an epoch means/represents. Can you add a comment + example about that on the `Height` godoc and also add comments on each of the fields in order to distinguish them?